src/Admin/Model/DoctrineRestServiceModel.php

```diff
@@ -187,10 +191,10 @@ public function getServiceManager()
     /**
      * Set service manager
      *
-     * @param ServiceManager $serviceManager
+     * @param ContainerInterface $serviceManager
      * @return DoctrineRestServiceModel
      */
-    public function setServiceManager(ServiceManager $serviceManager)
+    public function setServiceManager(ContainerInterface $serviceManager)
```

Comment:
yes, see above.

Ideally, though, these classes should be refactored to accept the specific dependencies they have via the constructor. I did that in a few places in zf-apigility-admin and one or two other modules while prepping for the v3 releases. The one place I kept it in zf-apigility-admin was in the autodiscovery feature — where it is required for determining if services exist.